packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -283,26 +321,73 @@ export function TransferPanelSummary({
     estimatedTotalGasFees
   } = gasSummary
 
-  const { app } = useAppState()
+  const {
+    app: { isDepositMode }
+  } = useAppState()
   const { ethToUSD } = useETHPrice()
   const { l1, l2 } = useNetworksAndSigners()
   const { parentLayer, layer } = useChainLayers()
 
   const { isMainnet } = isNetwork(l1.network.id)
 
+  const parentLayerGasFeeTooltipContent = useMemo(() => {
```

Comment:
i actually think that it'd be better to move the strings from `layerToGasFeeTooltip` object into this `useMemo` hook

what does everyone think?

Reply:
Imho, Since it's a static object, we can keep it as is currently.